level-2/README.md

```diff
@@ -0,0 +1,27 @@
+# karma-clone
+## Level 2
+The aim of this exercise is to add a new section to the homepage, and create a new page with an Order form.
+
+### Set up
+
+You can choose to add your code for Level 2 to the same branch you were using for Level 1, or to create a new 'level-2' branch. 
+
+# Instructions
+
+- You'll find the designs inside the 'level 2' folder.
```

Comment:
We've recently had an issue where students started re-creating designs that were in the repo files, rather than following the readme screenshot. I think there's no harm in leaving both in?

Reply:
Sounds like some students will need this info so let's leave it in then. Could we update `'level 2'` to `` `level-2` `` to match the folder name?